service/src/test/java/bio/terra/workspace/common/fixtures/WorkspaceFixtures.java

```diff
@@ -64,4 +79,22 @@ public static ApiCreateWorkspaceRequestBody createWorkspaceRequestBody() {
   public static String getUserFacingId(UUID workspaceId) {
     return String.format("user-facing-id-%s", workspaceId);
   }
+
+  public static ApiCreatedWorkspace createDefaultWorkspace(
```

Comment:
Thanks for creating. Can this go in MockMvcUtils? This is only for mock mvc tests. Also, call this createWorkspace.